addons/mail/static/tests/crosstab/crosstab.test.js

```diff
@@ -195,6 +195,7 @@ test("Message delete notification", async () => {
     const [partner] = pyEnv["res.partner"].read(serverState.partnerId);
     pyEnv["bus.bus"]._sendone(partner, "mail.message/delete", {
         message_ids: [messageId],
+        from_store: true,
```

Comment:
manual _sendone in tests are strongly deprecated, it would be best to move this part to mock server (in the same method as python) as we have diff on it